client/packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -153,6 +153,7 @@ const useStatusChangePlaceholderCheck = () => {
   const alert = useAlertModal({
     title: t('heading.cannot-do-that'),
     message: t('messages.must-allocate-all-lines'),
+    onOk: () => {},
```

Comment:
Shouldn't this better default to an empty method in `useAlertModal` when onOk is not specified to avoid similar errors in the future? or make onOk required?

Reply:
have defaulted - it's nice for the consumer and removes the issue. 🙏